e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,132 @@
+import forEach = require("cypress/types/lodash/forEach");
+
+interface Project {
+  id: string;
+  name: string;
+  type: string;
+}
+
+// assume 2.0 if not in CI. if you wish something different start cypress with
+// IAM_VERSION set to what you are testing.
+if (Cypress.env('IAM_VERSION') === undefined) {
+  Cypress.env('IAM_VERSION', 'v2.0');
+}
+
+const describeIAMV2P1 = Cypress.env('IAM_VERSION') === 'v2.1' ? describe : describe.skip;
+
+describeIAMV2P1('project management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const project1ID = `${cypressPrefix}-project1-${now}`;
+  const project1Name = `${cypressPrefix} project1 ${now}`;
+  const ruleID = `${cypressPrefix}-rule-${now}`;
+  const ruleName = `${cypressPrefix} rule ${now}`;
+
+  before(() => {
+    cy.adminLogin('/settings/projects').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminToken = admin.id_token;
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+    });
+  });
+
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+
+  afterEach(() => {
+    cy.saveStorage();
+  });
+
+
+  it('list projects', () => {
+    let projectIDs: string[];
```

Comment:
I might be missing something, but do we need this array in this scope?

In [line 51](https://github.com/chef/automate/pull/1248/files#diff-08c99676671eeb48b64efed39ef8b002R51), couldn't we do
```typescript
const projectIDs = response.body.projects.map((project: Project) => project.id);
projectIDs.forEach((id: string) => { // ...
```

or even
```typescript
response.body.projects.map((project: Project) => project.id)
  .forEach((id: string) => { // ...
```
?

Reply:
oo good point